editor/components/LightProps/DancerMode.tsx

```diff
@@ -85,16 +92,61 @@ const DancerMode = () => {
     setCurrentTab(newTab);
   };
 
-  const Tabs = Object.keys(displayParts).map((partType) => (
-    <Tab label={partType} value={partType} key={`property_tab_${partType}`} />
-  ));
+  const groupFilter = ([groupName, parts]: [
```

Comment:
Please add some comment for this function.

Reply:
done